lib/KeyCode.js

```diff
@@ -61,6 +61,7 @@ exports.KEY_W = 87;
 exports.KEY_X = 88;
 exports.KEY_Y = 89;
 exports.KEY_Z = 90;
+exports.KEY_LEFT_CMD = 91;
```

Comment:
While you are at it, can you also add `exports.KEY_RIGHT_CMD = 93`.

It is also the context menu button in Windows.

Reply:
Can you also add tests here: https://github.com/kabirbaidhya/keycode-js/blob/master/test/KeyCode.spec.js

And also update the TypeScript definition file: https://github.com/kabirbaidhya/keycode-js/blob/master/index.d.ts

Reply:
README and KeyCode.spec has been updated.

> Looks good to me. While you are at it, can you also add `exports.KEY_RIGHT_CMD = 93`.
> 
> It is also the context menu button in Windows keyboard layout.

So, you want to duplicate the 93 for two different KEY names?

Reply:
Yes, correct.

Reply:
Done

Reply:
@kabirbaidhya if you will merge this request, please make new release, I very need this changes in one of my projects.

Reply:
Sure, we will make a release.

One last change we need is to add the constants here: https://github.com/kabirbaidhya/keycode-js/blob/master/index.d.ts